tests/unit/core/test_contract_declaration.py

```diff
@@ -27,6 +27,25 @@ def test_abstract_contract(solc_binary_path) -> None:
     assert not slither.contracts[0].is_fully_implemented
 
 
+def test_concrete_contract(solc_binary_path) -> None:
+    solc_path = solc_binary_path("0.8.0")
+    slither = Slither(Path(CONTRACT_DECL_TEST_ROOT, "abstract.sol").as_posix(), solc=solc_path)
+    assert slither.contracts[0].is_fully_implemented
+
+    solc_path = solc_binary_path("0.5.0")
+    slither = Slither(
+        Path(CONTRACT_DECL_TEST_ROOT, "implicit_abstract.sol").as_posix(), solc=solc_path
+    )
+    assert slither.contracts[0].is_fully_implemented
+
+    slither = Slither(
+        Path(CONTRACT_DECL_TEST_ROOT, "implicit_abstract.sol").as_posix(),
+        solc_force_legacy_json=True,
+        solc=solc_path,
+    )
+    assert slither.contracts[0].is_fully_implemented
+
+
```

Comment:
It seems like a concrete contract should be tested here instead of abstract

Reply:
Sorry, this is my first open source pull request. Am I supposed to edit the checks or do anything else?

Reply:
No worries. Right now, we're asserting that this [abstract contract](https://github.com/crytic/slither/blob/dev/tests/unit/core/test_data/contract_declaration/abstract.sol) (`Path(CONTRACT_DECL_TEST_ROOT, "abstract.sol")`) is_fully_implemented. However, it will return `False` as it's abstract.

Since we want `is_fully_implemented` to return `True`, we could add a new solidity file [here](https://github.com/crytic/slither/blob/dev/tests/unit/core/test_data/contract_declaration/) called `concrete.sol` and then use `Path(CONTRACT_DECL_TEST_ROOT, "concrete.sol")`. Note, this contract should implement all of its functions and not use the `abstract` keyword.

Let me know if I can further help as we welcome contributions and appreciate it!